Makefile

```diff
@@ -29,9 +38,23 @@ req:
 install: prepare
 	$(PREFIX)pip install .
 
+# https://developer.atlassian.com/docs/getting-started/set-up-the-atlassian-plugin-sdk-and-build-a-project/install-the-atlassian-sdk-on-a-linux-or-mac-system#InstalltheAtlassianSDKonaLinuxorMacsystem-Homebrew
 install-sdk:
-	# https://developer.atlassian.com/docs/getting-started/set-up-the-atlassian-plugin-sdk-and-build-a-project/install-the-atlassian-sdk-on-a-linux-or-mac-system#InstalltheAtlassianSDKonaLinuxorMacsystem-Homebrew
+ifeq ($(PLATFORM), darwin)
```

Comment:
IMHO we shouldn't assume that people are running as a user with sudo or root access - i they don't have root / sudo access we can fail out and point them at a location to find out how to install the sdk.